V/VIC5/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+using BinaryBuilder
+
+name = "VIC5"
+version = v"0.1.1"
+
+# Collection of sources required to build
+sources = [
+    GitSource("https://github.com/CUG-hydro/VIC5.c.git",
+    "339d9666ce17e9638e16193c43637e453e3b03ef"), # v0.1.1
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/VIC5.c/vic
+
+mkdir -p "${libdir}"
+mkdir -p "${bindir}"
+
+target=${libdir}/libvic5_classic.${dlext} make dll -j${nproc}
+install_license ${WORKSPACE}/srcdir/VIC5.c/LICENSE.txt
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_cxxstring_abis(supported_platforms())
```

Comment:
Why `expand_cxxstring_abis`?  This looks like a pure C package, and the auditor doesn't say products contain `std::string` symbols.
```suggestion
platforms = supported_platforms()
```